common-utils/src/common/deployment/__init__.py

```diff
@@ -285,7 +259,11 @@ class FlowDeployment(BaseModel):
         {},
         description="Dictionary of parameter values to pass to this deployment's flow runs.",
     )
-    schedule: SCHEDULE_TYPES = Field(None, description="Schedule configuration for this deployment using Prefect Schedule Objects.")  # type: ignore
+    schedule: SCHEDULE_TYPES = (Field(None, description="Schedule configuration for this deployment using Prefect Schedule Objects."),)  # type: ignore
+    envars: list[FlowEnvar] = Field(
+        [],
+        description="List of plain text environment variables to pass to task run overrides",
+    )
```

Comment:
adding environment variable overrides to `FlowDeployment` models.  This enables the installation of python libs at runtime using the process described here: https://docs.prefect.io/latest/concepts/infrastructure/#installing-extra-dependencies-at-runtime